src/pymor/reductors/h2.py

```diff
@@ -705,14 +702,14 @@ def _poles_b_c_to_lti(poles, b, c):
     for i, pole in enumerate(poles):
         if pole.imag == 0:
             A.append(pole.real)
-            B.append(b[i].to_numpy().real)
-            C.append(c[i].to_numpy().real.T)
+            B.append(b[i].real)
+            C.append(c[i].real.T)
```

Comment:
I guess `c[i]` is now a 1D NumPy array.

```suggestion
            C.append(c[i].real[:, np.newaxis])
```